pcdet/models/detectors/detector3d_template.py

```diff
@@ -6,7 +6,7 @@
 from ...ops.iou3d_nms import iou3d_nms_utils
```

Comment:
Modify this file by considering `f2v` as `vfe`

Reply:
Done, moved `f2v` as a submodule of `ImageVFE`